public/js/icinga/loader.js

```diff
@@ -1057,9 +1056,6 @@
                         'Request to ' + url + ' has been aborted for ',
                         req.$target
                     );
-
-                    // Aborted requests should not be added to browser history
-                    req.addToHistory = false;
```

Comment:
Manual requests that have been aborted should still be added to history, as discussed yesterday. Though, scripted requests not. So set this to false, but only if `req.scripted`.